src/main/java/com/company/skyproeducationspring/controllers/ExamController.java

```diff
@@ -0,0 +1,25 @@
+package com.company.skyproeducationspring.controllers;
+
+import com.company.skyproeducationspring.services.exam.examiner.ExaminerService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/exam")
+public class ExamController {
+    private final ExaminerService examinerService;
+
+    public ExamController(ExaminerService examinerService) {
+        this.examinerService = examinerService;
+    }
+
+    @GetMapping(path = "/get")
+    public String getQuestions(int amount, Model model) {
```

Comment:
тут требовался PathVariable по заданию
```suggestion
    @GetMapping(path = "/get/{amount}")
    public String getQuestions(@PathVariable int amount, Model model) {
```
https://www.baeldung.com/spring-pathvariable

Reply:
Исправил